src/app/learn/accessibility/test/index.html

```diff
@@ -15,7 +15,7 @@
   </stache-page-anchor>
   
   <p>
-    SKY UX incorporates <a href="https://github.com/dequelabs/axe-core">aXe automated tests</a> into its <a routerLink="/learn/get-started/advanced/accessibility-analyzer">accessibility analyzer</a> for use in end-to-end (e2e) tests. aXe includes 50-60 rules covering a subset of the W3C's WCAG 2.0 success criteria. Include accessibility checks in e2e tests using the  <a routerLink="/learn/get-started/advanced/accessibility-analyzer">accessibility analyzer</a>, and ensure that all the rules pass.
```

Comment:
I think this doc change may expose a bigger SKY UX 3 issue. I think that when we talked about the testing library at standup yesterday, we discussed that it hasn't been migrated to SKY UX 3 yet. If so, it seems like a problem for accessibility unit tests, and I'm not sure what the guidance in the docs needs to be. ... @Blackbaud-SteveBrush, am I correct about the testing library not being available for SKY UX 3? If so, do we need to update the documentation to explain how to create accessibility unit tests in SKY UX 3? Or do we need to create a dev issue first to enable accessibility tests in SKY UX 3?

Reply:
Marking as resolved since you updated one link to the library and I just pushed changes to the other file that linked to it.